lib/logger.js

```diff
@@ -48,4 +48,12 @@ logger.error = function(error, metadata) {
   else childProcess.sendToParent('log', { level: 'error', message: message, metadata: metadata });
 };
 
+// Activity logger for users
+logger.logActivity = function(logData) {
+  var self = this;
+  userLogger.writeToCollection(logData, function(err) {
+    if (err) self.error(err);
```

Comment:
Actually, I think there's no need to involve `logger.js` at all. Just call `userLogger.writeToCollection` from `writeLog.js`, and if it errors you've already got `logger` in there.